src/a-runtime-console/kubernetes/service/devnamespace.scope.ts

```diff
@@ -1,27 +1,83 @@
-import {Injectable} from "@angular/core";
-import {ActivatedRoute, Router} from "@angular/router";
+import {Inject, Injectable} from "@angular/core";
+import {Headers, Http} from '@angular/http';
 import {Observable} from "rxjs";
-import {NamespaceScope} from "./namespace.scope";
+import {INamespaceScope} from "./namespace.scope";
+import {pathJoin} from "../model/utils";
+import {AuthenticationService} from "ngx-login-client";
+import {WIT_API_URL} from "ngx-fabric8-wit";
+import {Logger} from "ngx-base";
 
 /**
  * Defaults to using the Dev Space rather than the runtime environment
  * for things like BuildConfig or Builds
  */
 @Injectable()
-export class DevNamespaceScope extends NamespaceScope {
+export class DevNamespaceScope implements INamespaceScope {
   public namespace: Observable<string>;
 
-  constructor( activatedRoute: ActivatedRoute,  router: Router) {
-    super(activatedRoute, router);
+  private headers = new Headers({'Content-Type': 'application/json'});
+  protected userServicesUrl: string;
+  private currentNamespaceValue: string;
+
+  constructor(private logger: Logger,
+              private auth: AuthenticationService,
+              private http: Http,
+              @Inject(WIT_API_URL) apiUrl: string) {
+
+    console.log("========== DevNamespaceScope");
```

Comment:
Log statements should be removed before committing (this will fail the linter as well)